dbt/include/databricks/macros/metadata.sql

```diff
@@ -0,0 +1,32 @@
+{% macro databricks__get_relation_last_modified(information_schema, relations) -%}
+
+  {%- call statement('last_modified', fetch_result=True) -%}
+    {% if information_schema.is_hive_metastore %}
+        {%- for relation in relations -%}
+            select '{{ relation.schema }}' as schema,
+                    '{{ relation.identifier }}' as identifier,
+                    max(timestamp) as last_modified,
```

Comment:
just confirming: the output of a Databricks SQL query that includes a `max()` aggregation is never more than a single row, yes? That's why there is no `GROUP BY` present here?

Reply:
with the absence of group by, it will be a single row.